models/git_diff.go

```diff
@@ -418,6 +418,44 @@ func GetDiffRange(repoPath, beforeCommitID string, afterCommitID string, maxLine
 	return diff, nil
 }
 
+func GetRawDiff(repoPath, commitId string, diffType string) (string, error) {
```

Comment:
Can be simplified to `func GetRawDiff(repoPath, commitID, diffType string) (string, error) {`.